File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/RubyRing.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Accessory;

/* JADX INFO: loaded from: classes3.dex */
public class RubyRing extends Accessory {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.accessory_ruby_ring_name;
        this.idDescription = R.string.accessory_ruby_ring_description;
        this.idEffect = R.string.accessory_ruby_ring_effect;
        this.idImage = R.drawable.ruby_ring;
        this.price = 147L;
        this.initiative = true;
        this.defense = 10;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/RubyRing.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Accessory

class RubyRing : Accessory() {
    override fun configureProperties() {
        idName = R.string.accessory_ruby_ring_name
        idDescription = R.string.accessory_ruby_ring_description
        idEffect = R.string.accessory_ruby_ring_effect
        idImage = R.drawable.ruby_ring
        price = 147L
        initiative = true
        defense = 10
    }
}
